app/src/main/java/com/palone/planahead/screens/taskEdit/ui/segmentedButton/SegmentedRadioButton.kt

```diff
@@ -16,13 +18,23 @@ fun SegmentedRadioButton(
     Row(modifier = modifier, horizontalArrangement = Arrangement.Center) {
         fields.forEachIndexed { index, label ->
             SegmentedRadioButtonItem(
+                modifier = Modifier.weight(1f),
                 label = label,
                 isChecked = label == selectedField,
-                roundedCornerEnd = if (index == fields.size - 1) 10.dp else 0.dp,
-                roundedCornerStart = if (index == 0) 10.dp else 0.dp,
+                roundedCornerEnd = if (index == fields.size - 1) 20.dp else 0.dp,
+                roundedCornerStart = if (index == 0) 20.dp else 0.dp,
                 onValueChange = onValueChange
             )
         }
 
     }
+}
+
+@Preview
+@Composable
+fun Preview() {
+    SegmentedRadioButton(
+        fields = TaskType.values().toList(),
```

Comment:
Why are we using list instead of array? We can change the type of `fields` to `Array<T>` and that way we will be able to pass it without `toList()`.